stricher/week_02/GridClass.py

```diff
@@ -0,0 +1,75 @@
+class Grid:
+    def __init__(self, size: int, numRowsCols: int):
+        Grid.__check_params(size, numRowsCols, "Grid() - Invalid args")
+        self._size = size
+        self._numRowsCols = numRowsCols
+
+    @staticmethod
+    def __isInteger(val) -> bool:
+        import math
+        return math.floor(val) == val
+
+    @staticmethod
+    def get_cellSize(gridSize: int, numRowsCols: int) -> float:
+        numInteriorBorders = numRowsCols - 1
```

Comment:
Check out the style guide for variables definitions and other fun trivia: https://www.python.org/dev/peps/pep-0008/